catalogue/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM node:12.18.3
+
+WORKDIR /usr/src/app/webapp
+
+RUN apt-get update && apt-get install -y python python-pip rename
+RUN pip install awscli
+
+ADD ./.flowconfig ./.flowconfig
+ADD ./package.json ./package.json
+ADD ./yarn.lock ./yarn.lock
+ADD ./common ./common
+ADD ./flow-typed ./flow-typed
+
+ADD ./catalogue/webapp ./catalogue/webapp
+ADD ./content/webapp ./content/webapp
+
+RUN yarn setupCommon
+
+WORKDIR /usr/src/app/webapp/catalogue/webapp
+
+ENV BUILD_HASH=latest
+ENV BUNDLE_ANALYZE=both
+
+RUN yarn
+RUN yarn build
+
+WORKDIR /usr/src/app/webapp/catalogue/webapp/.dist
+RUN rename 's/^/catalogue./' *
```

Comment:
Interesting - why do we need to do this?

Reply:
This mirrors how the "next report" is built and deployed in circle: https://github.com/wellcomecollection/wellcomecollection.org/blob/7f97f6c03a2d455d32d0e6b95ecadcc0f3413cc4/.circleci/config.yml#L84